projects/hslayers/components/sidebar/sidebar.component.ts

```diff
@@ -133,7 +131,7 @@ export class HsSidebarComponent implements OnInit, OnDestroy {
       !this.HsLayoutService.sidebarExpanded;
     this.HsLayoutService.updPanelSpaceWidth();
     setTimeout(() => {
-      this.HsCoreService.updateMapSize();
+      this.HsEventBusService.updateMapSize.next();
```

Comment:
I know it was not changed in this PR, but I propose to follow a common convention and rename `updateMapSize` to `mapSizeUpdates`. What do you think?